### PR TITLE
GUI: Replace "Close" by "Close library"

### LINKED ISSUE #213: New Command Line Option -f
#### New Command Line Option -f

Offer a new command line option "-f" / "--filelinks" that links all bibtex-entries with the bibtex-named PDFs that are found in the respective directory. (this is ```ALT+f``` for a single entry in the GUI.)

##### Example application 

```jabref -f -i input.bib -o output.bib``` 

The new option will simplify batch-processing. 

#### Feature Generalization 

Enable ```ALT-f``` not only for single entry, but for a selected subset of entries (as ```CTRL-g```).

### PR DESCRIPTION
**Pull request by Luggas4you**
fixes https://github.com/JabRef/jabref-issue-melting-pot/issues/213

After:
image

### Mandatory checks
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/logic/importer/fetcher/MathSciNet.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public URL getURLForEntry(BibEntry entry) throws URISyntaxException, MalformedUR
 
     @Override
     public URL getURLForQuery(QueryNode luceneQuery) throws URISyntaxException, MalformedURLException, FetcherException {
-        URIBuilder uriBuilder = new URIBuilder("https://mathscinet.ams.org/mathscinet/search/publications.html");
+        URIBuilder uriBuilder = new URIBuilder("https://mathscinet.ams.org/mathscinet/publications-search");
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/logic/importer/fetcher/MathSciNet.java:79]: Those should not be part of this PR

File: src/main/java/org/jabref/gui/actions/StandardActions.java [hunk 2]
```diff
@@ -70,7 +70,7 @@ public enum StandardActions implements Action {
     EXPORT_SELECTED(Localization.lang("Export selected entries"), KeyBinding.EXPORT_SELECTED),
     CONNECT_TO_SHARED_DB(Localization.lang("Connect to shared database"), IconTheme.JabRefIcons.CONNECT_DB),
     PULL_CHANGES_FROM_SHARED_DB(Localization.lang("Pull changes from shared database"), KeyBinding.PULL_CHANGES_FROM_SHARED_DATABASE),
-    CLOSE_LIBRARY(Localization.lang("Close"), Localization.lang("Close the current library"), IconTheme.JabRefIcons.CLOSE, KeyBinding.CLOSE_DATABASE),
+    CLOSE_LIBRARY(Localization.lang("Close library"), Localization.lang("Close the current library"), IconTheme.JabRefIcons.CLOSE, KeyBinding.CLOSE_DATABASE),
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/actions/StandardActions.java:73]: you will need to fix this in en.properties as well.  file https://devdocs.jabref.org/code-howtos/localization.html

File: src/main/java/org/jabref/logic/importer/fetcher/MathSciNet.java (hunk 1 above)
Comment (Luggas4you) [reply src/main/java/org/jabref/logic/importer/fetcher/MathSciNet.java]: Yes, I fixed it. It happened twice now, but I don't know why. Those changes were not displayed...

**Comment by ThiloteE** [comment]
Please add following info to the pull-request comment:

1. Link the issue from melting pot with github syntax similar to "fixes #10043". It is unclear which issue you mean. Also, melting-pot issues are hidden from view for non-admins.
2. If it is a GUI change, please add pictures from "before" and "after".

File: src/main/java/org/jabref/gui/actions/StandardActions.java (hunk 2 above)
Comment (Luggas4you) [reply src/main/java/org/jabref/gui/actions/StandardActions.java:73]: There is this one: 
image

So I just delete this one:
image
?

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/actions/StandardActions.java:73]: Add a new one with close library. I guess Close is used in other parts of the GUI as well
Just run LocalizationConsistencyTest it will tell you

**Comment by Siedlerchr** [comment]
and don't forget a changelog entry :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 
 ### Changed
 
+- We replaced "Close" by "Close library" and placed it after "Save all" in the File tab. [#10043] (https://github.com/JabRef/jabref/pull/10043)
```
Comment (calixtus) [inline CHANGELOG.md:29]:
```suggestion
- We replaced "Close" by "Close library" and placed it after "Save all" in the File menu. [#10043] (https://github.com/JabRef/jabref/pull/10043)
```

Comment (Siedlerchr) [reply CHANGELOG.md:29]: and no space after the `[]`and the round `()`ones

**Comment by Luggas4you** [comment]
Applied all the suggestions. Code should look clean now.

**Review by koppor — APPROVED** [review]

**Review by calixtus — APPROVED** [review]
